deploy/hosted-registry/devfile-registry.yaml

```diff
@@ -19,7 +19,7 @@ objects:
       app: devfile-registry
     name: devfile-registry
   spec:
-    replicas: 1
+    replicas: ${{REPLICAS}}
```

Comment:
I am using zsh, seems like `${{REPLICAS}}` doesn't work for zsh to interpret the env var, should we use `${REPLICAS}` instead? It should work for all shells.

Reply:
Gah, lemme try zsh, still on bash

Reply:
Updated to use single curly brackets, good find Jingfu 👍